packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -25,43 +25,91 @@ import { IL2StandardBridge } from "interfaces/L2/IL2StandardBridge.sol";
 /// @notice Reusable test initialization for `L2StandardBridge` tests.
 contract L2StandardBridge_TestInit is CommonTest {
     /// @notice Sets up expected calls and emits for a successful ERC20 withdrawal.
-    function _preBridgeERC20(bool _isLegacy, address _l2Token) internal {
-        // Alice has 100 L2Token
-        deal(_l2Token, alice, 100, true);
-        assertEq(ERC20(_l2Token).balanceOf(alice), 100);
+    function _expectERC20WithdrawalCalls(
+        uint256 _amount,
+        uint32 _minGasLimit,
+        address _to,
+        bool _isLegacy,
+        address _l2Token
+    )
+        internal
+    {
+        deal(_l2Token, alice, _amount, true);
+        assertEq(ERC20(_l2Token).balanceOf(alice), _amount);
```

Comment:
It doesn't make sense to have this `deal` inside a `_expectERC20WithdrawalCalls` function.